src/resources/Sources/SourcesSubInterfaces/Permission.ts

```diff
@@ -40,6 +40,57 @@ export interface DocumentPermissionModel {
     state?: DocumentPermissionState;
 }
 
+export interface SecurityIdentityInfoModel {
+    key?: string;
+    value?: string;
+}
+
+export interface DetailedSecurityIdentityModel {
+    name?: string;
+    provider?: string;
+    type?: string;
```

Comment:
Is just any string accepted for those types, or are there specific expected values? Might be worth adding new enums (like those imported above).

Reply:
Changed it to `PermissionIdentityType`.